### PR TITLE
Add powershell script to install the CLI

### PR DESCRIPTION
**Pull request by jwood803**

### CODE REVIEW

**Comment by jwood803** [comment]
Should fix https://github.com/exercism/exercism.io/issues/2272

File: app/pages/process/installing-the-cli-on-windows.md [hunk 1]
```diff
@@ -78,6 +78,24 @@ The following instructions will append the location of the exercism.exe file to
 
 ## Using the Chocolatey Package Manager for Windows
 **NOTE:** You can find more information about using Chocolatey at the [Chocolatey site](https://chocolatey.org/).
+
+### Using PowerShell
+Windows ships with a powerful scripting language called [PowerShell](https://en.wikipedia.org/wiki/Windows_PowerShell). With this it's fairly easy to get started.
+
+Run the following script and you're ready to go. This downloads Chocolatey, sets the environment path variable, and installs the exercism CLI.
+
```
Comment (kytrinyx) [inline app/pages/process/installing-the-cli-on-windows.md:86]: How do you get to a place where you can run it?

Comment (jwood803) [inline app/pages/process/installing-the-cli-on-windows.md:86]: Ahhh, good point! I'll update it.

**Comment by jwood803** [comment]
This should do it. I'll attempt to squash these.

**Comment by kytrinyx** [comment]
Sweet, thanks.

**Pull request merged**